packages/content/src/Content.tsx

```diff
@@ -27,7 +27,7 @@ export function Content({
 				width="100%"
 				maxWidth={tokens.maxWidth.container}
 				paddingY={paddingYMap[spacing]}
-				paddingX={{ xs: 1, md: 2 }}
+				paddingX={tokens.containerPadding}
```

Comment:
This is fine for now but it's starting to feel like we may be getting into the territory of `component-tokens`. Would be good to consider how we want to handle those.